helpers/campaignLimiting.js

```diff
@@ -0,0 +1,20 @@
+const BN = require('bn.js')
+
+function shouldWeLimitCampaigns (filteredCampaigns, limit) {
```

Comment:
the way this is called doesn't take `filteredCampaigns` - it must check if the campaigns that do have a non-zero `status.lastApprovedBalances[publisherAddr]` are more than the limit

but it just takes in all the campaigns (it seems)